lambda/main.go

```diff
@@ -57,6 +57,12 @@ func Handler(ctx context.Context, evt json.RawMessage) (string, error) {
 	quiet := quietString == "1" || quietString == "true"
 	timeout := os.Getenv("BUILDKITE_AGENT_METRICS_TIMEOUT")
 
+	debugEnvVar := os.Getenv("BUILDKITE_AGENT_METRICS_DEBUG")
+	debug := debugEnvVar == "1" || debugEnvVar == "true"
```

Comment:
I think, one day, we should use [strconv.ParseBool](https://pkg.go.dev/strconv#ParseBool) for all these.

Reply:
I kinda wish it were stricter, if it only accepted "true" and "false" and errored on everything else, I would be more happy.